composer.json

```diff
@@ -22,7 +22,8 @@
         "phpstan/phpstan": "1.10.47 || 1.4.10",
         "phpunit/phpunit": "^9.6 || ^7.5",
         "psr/container": "^2 || ^1",
-        "react/promise-timer": "^1.10"
+        "react/promise-timer": "^1.10",
+        "psr/http-server-middleware": "^1"
```

Comment:
This should(tm) only be a requirement for the tests to run, so I added it as dev-dependency

Reply:
Seems about right, we'd rather add a new dev-dependencies instead of a new requirement, as this only affects development and not all users 👍